azurekeyvault-flexvolume/keyvaultFlexvolumeAdapter.go

```diff
@@ -121,8 +121,11 @@ func (adapter *KeyvaultFlexvolumeAdapter) initializeKvClient() (*kv.BaseClient,
 	return &kvClient, nil
 }
 
-func wrapObjectTypeError(err error, objectType string, objectName string, objectVersion string) error {
-	return errors.Wrapf(err, "failed to get objectType:%s, objectName:%s, objectVersion:%s", objectType, objectName, objectVersion)
+// azure-sdk-for-go returns some errors with \r\n in the body
+// kubernetes errors out with "invalid character '\r' in string literal", if we don't sanitise it first
+func sanitisedError(err error, objectType string, objectName string, objectVersion string) error {
+	sanitisedErr := strings.Replace(err.Error(), "\\r\\n", " ", -1)
```

Comment:
```suggestion
	sanitisedErr := strings.Replace(err.Error(), "\\", " ", -1)
```

Reply:
Got this error when I tested this change:
`MountVolume.SetUp failed for volume "test" : invalid character 'g' in string escape code`
Seems the sdk is returning `Operation: \"get\" Caller` as part of the error message. Let's replace all `\` escape characters to avoid any additional `\` in the message returned by server side.

Reply:
Ah nice, I didn’t test with an access policy but no get access

Reply:
Ah nice, I didn’t test with an access policy but no get access